apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/commons/TransformerConstants.java

```diff
@@ -357,6 +357,9 @@
    */
   public static final String HL7_RACE_UNKNOWN_DISPLAY = "Unknown";
 
+  /** System for encoding UNK/null values */
+  public static final String CODING_V3_NULL = "http://terminology.hg/CodeSystem/v3-NullFlavor";
```

Comment:
There is a typo here.  Should be `.org` not `.hg`:

http://terminology.hl7.org/CodeSystem/v3-NullFlavor